monitor-wrapper.py

```diff
@@ -0,0 +1,74 @@
+from prometheus_client import start_http_server, Counter
```

Comment:
Since the rekor-monitor is a Go application, do we really need a Python wrapper? I know that we are considering migrating the segment backup job to the Go language to avoid Python support. This would bring us Python support back. I was thinking about extending the monitor bin rather than wrapping it with another CLI. WDYT?